### PR TITLE
Add comments and minor renames to peerwg

### PR DESCRIPTION
**Pull request by kylecarbs**

### CODE REVIEW

File: agent/agent.go [hunk 1]
```diff
@@ -47,30 +47,30 @@ const (
...
+	UploadWireguardKeys    UploadWireguardKeys
 	ListenWireguardPeers   ListenWireguardPeers
 	ReconnectingPTYTimeout time.Duration
 	EnvironmentVariables   map[string]string
 	Logger                 slog.Logger
 }
 
 type Metadata struct {
-	Addresses            []netaddr.IPPrefix `json:"addresses"`
+	WireguardAddresses   []netaddr.IPPrefix `json:"addresses"`
```
Comment (coadler) [inline agent/agent.go:58]:
```suggestion
	WireguardAddresses   []netaddr.IPPrefix `json:"wireguard_addresses"`
```

File: agent/agent.go [hunk 2]
```diff
@@ -114,8 +114,8 @@ type agent struct {
 	sshServer     *ssh.Server
 
 	enableWireguard      bool
-	wg                   *peerwg.WireguardNetwork
-	postKeys             PostKeys
+	network              *peerwg.Network
+	postKeys             UploadWireguardKeys
```
Comment (coadler) [inline agent/agent.go:118]:
```suggestion
	uploadWireguardKeys             UploadWireguardKeys
```
formatting shit but u get the point

File: peer/peerwg/peermessage.go [hunk 3]
```diff
@@ -10,35 +10,35 @@ import (
...
 	if idx == -1 {
 		return false, xerrors.Errorf("invalid peer message, no separator")
 	}
 
 	return bytes.Equal(agentID, msg[:idx]), nil
 }
 
-func (pm *WireguardPeerMessage) UnmarshalText(text []byte) error {
-	sp := bytes.Split(text, []byte{peerMessageSeparator})
+func (pm *Handshake) UnmarshalText(text []byte) error {
```
Comment (coadler) [inline peer/peerwg/peermessage.go:40]: should rename these method receivers too

**Review by coadler — APPROVED** [review]

File: peer/peerwg/peermessage.go (hunk 3 above)
Comment (kylecarbs) [reply peer/peerwg/peermessage.go:40]: true will do

**Pull request merged**